tests/cases/fourslash/autoImportTypeImport4.ts

```diff
@@ -0,0 +1,111 @@
+/// <reference path="fourslash.ts" />
+
+// @verbatimModuleSyntax: true
+// @target: esnext
+
+// @Filename: /exports1.ts
+//// export const a = 0;
+//// export const A = 1;
+//// export const b = 2;
+//// export const B = 3;
+//// export const c = 4;
+//// export const C = 5;
+//// export type x = 6;
+//// export const X = 7;
+//// export const Y = 8;
+//// export const Z = 9;
+
+// @Filename: /exports2.ts
+//// export const d = 0;
+//// export const D = 1;
+//// export const e = 2;
+//// export const E = 3;
+
+// @Filename: /index0.ts
+//// import { A, B, C } from "./exports1";
+//// a/*0*/;
+//// b;
+
+// @Filename: /index1.ts
+//// import { A, B, C, type Y, type Z } from "./exports1";
+//// a/*1*/;
+//// b;
+
+// @Filename: /index2.ts
+//// import { A, a, B, b, type Y, type Z } from "./exports1";
+//// import { E } from "./exports2";
+//// d/*2*/
+
+// addition of correctly sorted type imports should not affect behavior as shown in autoImportSortCaseSensitivity1.ts
+goTo.marker("0");
+verify.importFixAtPosition([
+    `import { A, B, C, a } from "./exports1";\na;\nb;`,
+    `import { A, B, C, b } from "./exports1";\na;\nb;`,
+],
+    /*errorCode*/ undefined,
+    { organizeImportsTypeOrder : "last" });
+verify.importFixAtPosition([
+    `import { a, A, B, C } from "./exports1";\na;\nb;`,
+    `import { A, b, B, C } from "./exports1";\na;\nb;`

```

Comment:
Just so I'm clear, these are alternatives such that both are acceptable? For this position, shouldn't it be deterministic and one always wins?

Reply:
There's two results because there are two different codefixes that show up, one adds `a`, and the other adds `b`

Reply:
Ohh I cannot read. Thanks.